ompi/mca/osc/rdma/osc_rdma_accumulate.c

```diff
@@ -192,9 +192,9 @@ static int ompi_osc_rdma_fetch_and_op_cas (ompi_osc_rdma_sync_t *sync, const voi
         new_value = old_value;
 
         if (&ompi_mpi_op_replace.op == op) {
-            memcpy ((void *)((intptr_t) &new_value + offset), origin_addr + dt->super.true_lb, extent);
+            memcpy ((void *)((intptr_t) &new_value + offset), (void *)((intptr_t) origin_addr + dt->super.true_lb), extent);
```

Comment:
Technically this should be ptrdiff_t.

Reply:
Ah.  This is a cherry-pick from master.  I can make a PR for master to cherry pick back to here.

Reply:
BTW, are you saying it should be this:

```suggestion
            memcpy ((void *)((intptr_t) &new_value + offset), (void *)((ptrdiff_t) origin_addr + dt->super.true_lb), extent);
```